### ISSUE TITLE
Kitchen counters

### ISSUE THREAD

**Issue by iteles** [dwyl/home#192]
Our kitchen counters have to be custom made because of the layout of the piping in the house.
This means we have to have one extra-wide counter.

This was the proposal from Ikea, but their counters top out at 2,95m long so their would have to have a cut in them.
![IMG_9689](https://user-images.githubusercontent.com/4185328/157707729-7d1a0383-7241-4834-ba53-bc58b24c0d16.jpg)

Opening this issue for a little extra research before we go with these.

**Comment by nelsonic** [comment]
@iteles at this stage we're only interested in the L-shaped section. 
We would not need to make any cuts because the total length of the counter against the wall is less than 2950mm.
If we could get it _done_ it would make a _huge_ difference to our lives. 
We can revisit the rest of the kitchen at a later date even at increased marginal cost.

**Comment by iteles** [comment]
I spoke to Rafael from [KStar kitchens](http://www.kstar.pt/index) in Braga and they are open from 10-13h00 and then from 15h00-19h00.

To give us an idea:
+ Cheapest is Silestone for 200 m2 (but this is covered in a resin so hot things can't go on it)
+ More expensive is Dekton which is 400m2 and goes much higher as well

7 mins from the house:
Rua Cidade do Porto, 197
4705-087 Braga, Portugal

Tel./Fax
253 117 072
Telmóvel
910 506 606

<hr/>

These guys also do the same materials as well as [`Laminam`](https://www.laminam-portugal.pt/) but when I asked him for prices he basically said there was nothing he could tell me, they'd have to send the measurements to the manufacturers for an orçamento:
https://cookit.com.pt/index.html

10 mins from the house:

Rua do Quintão, 39, r/c
Frossos
4700-153 Braga

### I think these guys will all basically have the same manufacturers so there's no point looking for more.

So I spoke to [these guys](https://delicat.com.pt/#/contactos) and they said they make their own kitchens but they always get their counters from another supplier. 
I asked the lady for a recommendation for a `marmorista` and she suggested [Alpstone](https://www.alpstone.com.pt/en/home/).

If we want a composite stone, they are open tomorrow 08h30 - 12h30 and we can go visit to see what they offer.

**Comment by nelsonic** [comment]
We have **3 weeks** to make a final decision on the **`color`** we are going with for the counter tops. ⏳ 
I'm still totally "torn" between the Cement look and the White ("clean") look ... 🙄 
This is a pretty _expensive_ decision ... 💸 
@iteles have you had any further thoughts on the topic? 💭

**Comment by nelsonic** [comment]
In #197 @iteles shared a picture of **Sadia** (Pick Up Limes) https://www.pickuplimes.com/about kitchen:

![pick-up-limes-kitchen](https://user-images.githubusercontent.com/4185328/162977100-3b5e1d46-bbb6-4942-96b8-c18d9cf1015d.png)

![image](https://user-images.githubusercontent.com/194400/163272915-fa756131-c4e4-4daa-b3b4-f886e6b0411b.png)

The lighter counter and darker backsplash/background improves contrast in photos/videos.

**Comment by nelsonic** [comment]
@iteles we'll need to decide this soon. 
Do you want to look at Sample at Leroy Merlin, Casa Peixoto & IKEA to make a final decision e.g: Monday evening?

**Comment by iteles** [comment]
We are having the Ikea measurements done today seeing as we'd already paid for that portion and will get the final quote in a day or two.

I still prefer the Leroy Merlin Dekton we saw because it's matte and generally has a nicer colour in my view.
Let's wait until we get the new Ikea quote and if the Leroy Merlin one isn't _much_ more expensive, I vote we get that one.

**Comment by nelsonic** [comment]
Agreed. Leroy has _many_ more options (decision fatigue?) for counter tops than IKEA.
We should axsk to borrow one of the samples of Dekton and take it with us to IKEA before making the final decision.

**Comment by iteles** [comment]
Things to be aware of:
+ Ikea requires a 3.5cm overhang from the metal frame onwards in order to protect the cupboards
+ We will require significantly more grooves for the drainage board on the L-shaped kitchen (9 instead of 5)

**Comment by nelsonic** [comment]
@iteles also the measurement guy noted that the induction hob thickness is more than thickness of the counter top:

![2263DBD2-9ACB-486D-8030-D2AC6E4391D4](https://user-images.githubusercontent.com/194400/166494055-abe84271-0a0b-4603-9bae-36ce54de4e47.jpeg)

so in order for *this* hob to fit, we would need to lower the oven by (at least) 5mm.
Want to call up IKEA and splain to them? 🙄

**Comment by nelsonic** [comment]
@iteles have you been able to follow up with IKEA to figure out when they can adjust the counters and/or reposition the oven to accomodate the counter tops? 💭

**Comment by iteles** [comment]
They won't reposition the ovens because the ovens are not IKEA, so that's on us unfortunately.

We're waiting for the call from LM to get the final measurements sorted asap so that we can move forward with the counters that have been paid for. 50% paid already and another 50% to be paid once the measurements are confirmed.

There is a shortage of the Dekton stone we chose so we're also awaiting news on availability.

**Comment by nelsonic** [comment]
lol. supply "shortages" of manufactured products are 100% fabricated to drive up prices. 🤦‍♂️ 

How Dekton is made (EN) | Cosentino: https://youtu.be/R2FnhX6Fa4U
[![image](https://user-images.githubusercontent.com/194400/168471343-c9b0d1db-1410-4b53-860c-1ad7d22867d3.png)](https://youtu.be/R2FnhX6Fa4U)

There is _zero_ shortage of the raw materials and their state-of-the-art factory 
has been designed to eliminate production bottlenecks. 💭

**Comment by nelsonic** [comment]
I'm saying that it _sounds_ like a _manufactured_ "shortage" regardless of where in the supply chain ... 🙄 
The roads from Italy aren't closed. The trucks aren't stopped. The drivers aren't on _strike_. What's the _shortage_?! 🤷‍♂️ 
(rhetorical question. it's artificial scarcity.)

**Comment by iteles** [comment]
I spoke to Cláudia from LM just now:
+ The marmorista should have already called us to book measurements, she is following up with him today
+ They will receive a shipment from the factory on Friday but don't know whether the Dekton we have chosen is a part of the shipment. They will find out on Friday and we can go from there.

**Comment by nelsonic** [comment]
That's great. Let's see if they actually get back to us this week. 🤞  ⏳

**Comment by nelsonic** [comment]
@iteles does the second instalment of the kitchen counters need to paid *before* they deliver+install the counters? If so should I do it *today*? 🤷🏼‍♂️

**Comment by iteles** [comment]
The answer to that is I don't know but to be on the safe side I would do it first thing tomorrow morning 👍

**Comment by nelsonic** [comment]
Ok I will pay it as a shallow-work break after I’ve got some work-work done tomorrow morning.

**Comment by nelsonic** [comment]
FYI: Just in case Leroy calls you to axsk if you've paid.  ✅ 💸  
Details uploaded to Google Drive as per uzhe. 📁
